src/nodes.coffee

```diff
@@ -9,6 +9,12 @@ Error.stackTraceLimit = Infinity
 {RESERVED, STRICT_PROSCRIBED} = require './lexer'
 iced = require 'iced-runtime'
 
+# At this stage of development of this branch, it's easy to mess up
+# runtimes. The one in node_modules/, which is used by default, will
+# probably be incorrect.
+if not iced.const.iterator
+  throw new Error 'Wrong iced runtime!'
```

Comment:
Let me know when you update the branch to reflect this, I'll repull and then demo, and then do the merge. Thanks!